.github/workflows/reusable_kernel_tests.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
           run_id: "id-${{ github.run_id }}"
           output_dir: "~/ansible_output_${{ github.run_id }}"
           repos:
-            libs: {name: "falcosecurity-libs", repo: "https://github.com/${{ github.event.inputs.libsrepo }}.git", version: "${{ github.event.inputs.libsversion }}"}
+            libs: {name: "falcosecurity-libs", repo: "https://github.com/${{ inputs.libsrepo }}.git", version: "${{ inputs.libsversion }}"}
```

Comment:
Tested here: https://github.com/falcosecurity/libs/actions/runs/6505305275/job/17668682455
Now they are correctly expanded:
```
  libs: {name: "falcosecurity-libs", repo: "https://github.com/falcosecurity/libs.git", version: "master"}
```

Reply:
The other syntax was the old syntax to access `workflow_dispatch` inputs, that was not working instead with `workflow_call`.
See https://github.blog/changelog/2022-06-10-github-actions-inputs-unified-across-manual-and-reusable-workflows/ for reference.